wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ZMConversation+Status.swift

```diff
@@ -356,7 +356,7 @@ final class SecurityAlertMatcher: ConversationStatusMatcher {
            let messageText = data.messageText {
             text = messageText
         } else {
-            text = "conversation.status.poll.default".localized
+            text = ""
```

Comment:
`conversation.status.poll.default".localized` doesn't exist so for now I just removed it.